miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/ListTablesIT.java

```diff
@@ -291,6 +292,7 @@ public void testListPoolsPageSetup() throws Exception {
     testTabbedPageSetup(ListTarget.POOLS, poolsColumns);
   }
 
+  @Ignore // Broken since warnings are currently not sorted.
```

Comment:
These two sorting tests are important. In the `list_x.js` files, set `bSortable` to false and make sure it's not on the sortable columns list.